src/xcode/ENA/ENA/Resources/Assets/Assets.xcassets/Illustrations/Illu_TraceLocations_HomeCard.imageset/Contents.json

```diff
@@ -0,0 +1,25 @@
+{
+  "images" : [
+    {
+      "filename" : "Illu_TraceLocation_HomeCard.pdf",
```

Comment:
Not really sure why you added this a second time, seems to be identical to the existing one?

Reply:
removed the duplicate illustration with commit 8a87c4a